src/bp-forums/groups.php

```diff
@@ -289,13 +554,63 @@ public function edit_screen( $group = false ) {
 					<h4 class="bb-section-title"><?php esc_html_e( 'Connected Forum', 'buddyboss' ); ?></h4>
 					<p class="bb-section-info"><?php esc_html_e( 'Only site administrators can reconfigure which forum belongs to this group.', 'buddyboss' ); ?></p>
 					<?php
+						// Fiter forum query for set the selected forum (Group associate forum).
+						add_filter( 'posts_where', array( $this, 'update_forum_dropdown_qeury' ), 10, 2 );
```

Comment:
`update_forum_dropdown_qeury` here `query` is spelled wrong. please check it.

Reply:
Fixed https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/a51489618f8f635d3ef6a0dc7d8b1a511d1be484